### DISCUSSION TITLE
🚀 Add note / validation to "New Folder" diaglog in Files area to prevent users trying to create folders with spaces in file name

### DISCUSSION THREAD

**Discussion by jcogs-design** [ExpressionEngine/ExpressionEngine#3818] [Feature Requests]
### Suggest an idea for ExpressionEngine

The dialog that appears does not indicate that the folder names should not include <insert excluded characters here> nor does it validate the input in the dialog - so user puts in "Illegal Name" hits Create and gets an error message on return page and no folder.
![image](https://github.com/ExpressionEngine/ExpressionEngine/assets/13821249/ee9734a2-4398-46fa-bcc8-44f7eb38d107)

**Comment by Yulyaswan** [comment]
PR https://github.com/ExpressionEngine/ExpressionEngine/pull/4013
